students/Dustin_L/lesson07/html_render.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+"""HTML Render Module
+
+This module contains all of the functions for the HTML Render module
+"""
+
+
+class Element(object):
+    """Element class for rendering an HTML element"""
+    tag = ''
```

Comment:
It's not necessary to define an empty attribute in Python in order to establish a shared interface for subclasses, though I can see how the instructions for the assignment lead you that way here. But I think the `tag` attribute could be omitted and defined in each subclass.